src/documents/IoCReferences.md

```diff
@@ -169,13 +183,13 @@ fluid.defaults("fluid.prefs.enactor.tableOfContents", {
 
 The example below includes several IoC references. All of them are inside a subcomponent declaration and all include `{controllers}`, which in this case is a reference to the parent component. Specifically:
 
-* `{controllers}.model` and `{controllers}.applier` are references to the model and applier that are members of the parent component;
+* `{controllers}.model` is references to the model that is a members of the parent component - note that this reference sets up a permanent [model relay](ModelRelay.md) between these two models;
```

Comment:
There's a grammatical issue that has me confused about what this sentence is saying.